web/styles.css

```diff
@@ -232,6 +232,35 @@ input:disabled+label {
     opacity: 0.5;
 }
 
+.inspector-tree {
```

Comment:
I'd create an inspector specific style sheet, `lib/inspector/inspector.css`, and move things into there that won't be shared. We do this for the debugging and logging pages, just so this css file doesn't grow unbounded.

Reply:
done